_data/contribute.yml

```
@@ -17,7 +17,7 @@
       questions:
         - tag: "ask-a-question"
           question: "How do I ask a question?"
-          answer: "<p>For general questions about methods, analysis, and code, the best place to maximize the number of eyes on your issue is by way of <a href='http://www.neurostars.org' target='_blank'>neurostars</a>. Questions about specific packages are best to post as issues on the repository itself, a link to which you can find on any of the package pages. To be added as a contributor or ask a question about the site, you can <a href='http://neuroimaging.tumblr.com/ask'> ask us a question</a>.</p>"
+          answer: "<p>For general questions about methods, analysis, and code, the best place to maximize the number of eyes on your issue is by way of <a href='https://neurostars.org/' target='_blank'>Neurostars</a>. Questions about specific packages are best to post as issues on the repository itself, a link to which you can find on any of the package pages. To be added as a contributor or ask a question about the site, you can reach us <a href='mailto:neuroimaging@python.org'>by email</a>. Although you can ask your question through the <a href="http://neuroimaging.tumblr.com/ask">neuroimaging tumblr ask space</a> note that it gets less attention than the previous resources.</p>"
```

```suggestion
          answer: "<p>For general questions about methods, analysis, and code, the best place to maximize the number of eyes on your issue is by way of <a href='https://neurostars.org/' target='_blank'>Neurostars</a>. Questions about specific packages are best to post as issues on the repository itself, a link to which you can find on any of the package pages. To be added as a contributor or ask a question about the site, you can reach us <a href='mailto:neuroimaging@python.org'>by email</a>. Although you can ask your question through the <a href='http://neuroimaging.tumblr.com/ask'>neuroimaging tumblr ask space</a> note that it gets less attention than the previous resources.</p>"
```
You can't have double quotes inside the double quotes to distinguish the field.

---

:man_facepalming: Thanks @vsoch . Changed for simple quotes in a3bdd79.